tests/tests_pytorch/accelerators/test_registry.py

```diff
@@ -11,55 +11,7 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
-from pytorch_lightning import Trainer
-from pytorch_lightning.accelerators import Accelerator, AcceleratorRegistry
-
-
-def test_accelerator_registry_with_new_accelerator():
```

Comment:
Moved this test. It does not need to be duplicated because what is tested here is independent of where the registry is used